### PR TITLE
Auto-use Max Elemental Damage

### PR DESCRIPTION
**Pull request by GreatOldOne**
Activate Max Elemental Damage whenever you've it, it's not cooling down and no Max Elemental Damage is activate in the lane.

### CODE REVIEW

File: autoPlay.user.js [hunk 1]
```diff
@@ -1170,6 +1171,20 @@ function useTreasureIfRelevant() {
...
 }
 
+function useMaxElementalDmgIfRelevant() {
+	// Check if Max Elemental Damage is purchased
+	if (hasItem(ITEMS.MAXIMIZE_ELEMENT)) {
+		if (isAbilityCoolingDown(ITEMS.MAXIMIZE_ELEMENT) || !canUseItem(ITEMS.MAXIMIZE_ELEMENT) || isAbilityActive(ITEMS.MAXIMIZE_ELEMENT)) {
+			return;
+		}
+		else {
+			// Max Elemental Damage is purchased, cooled down, and needed. Trigger it.
```
Comment (iakat) [inline autoPlay.user.js:1181]: Please use `getActiveAbilityLaneCount` to check for max elemental already active in current lane.

**Comment by GreatOldOne** [comment]
Done!

File: autoPlay.user.js [hunk 2]
```diff
@@ -1170,6 +1171,20 @@ function useTreasureIfRelevant() {
 	}
 }
 
+function useMaxElementalDmgIfRelevant() {
+	// Check if Max Elemental Damage is purchased
+	if (hasItem(ITEMS.MAXIMIZE_ELEMENT)) {
+		if (isAbilityCoolingDown(ITEMS.MAXIMIZE_ELEMENT) || !canUseItem(ITEMS.MAXIMIZE_ELEMENT) || getActiveAbilityLaneCount(ITEMS.MAXIMIZE_ELEMENT) > 0) {
+			return;
+		}
+		else {
```
Comment (iakat) [inline autoPlay.user.js:1180]: Get rid of this `else`. It's useless after a `return`.

**Comment by iakat** [comment]
Can you squash your commits?

**Comment by GreatOldOne** [comment]
Yes, I was just following the semantics style of the other functions, but alright.

**Comment by GreatOldOne** [comment]
I'll close this request and send another one with everything,

**Pull request closed without merging**